extensions/wikia/SpecialContact2/SpecialContact.body.php

```diff
@@ -762,16 +753,3 @@ private function validateUserName( $userName ) {
 		return true;
 	}
 
-	/**
-	 * Check if the CloseMyAccount extension is enabled and supported in the
-	 * current language.
-	 *
-	 * @return boolean True if CloseMyAccount is enabled and supported in the
-	 *                 current language, false otherwise
-	 */
-	private function isCloseMyAccountSupported() {
-		global $wgContLang, $wgEnableCloseMyAccountExt, $wgSupportedCloseMyAccountLang;
-		return !empty( $wgEnableCloseMyAccountExt )
-				&& in_array( $wgContLang->getCode(), $wgSupportedCloseMyAccountLang );
```

Comment:
Can we remove `wgSupportedCloseMyAccountLang` from the config repo as well as cleanup?